api/src/crons/JeVeuxAiderDaily.js

```diff
@@ -0,0 +1,269 @@
+require("dotenv").config({ path: "./../../.env-staging" });
+
+const fetch = require("node-fetch");
+require("../mongo");
+const { capture } = require("../sentry");
+const { JVA_API_KEY } = require("../config");
+const slack = require("../slack");
+const MissionModel = require("../models/mission");
+const StructureModel = require("../models/structure");
+const ReferentModel = require("../models/referent");
+
+const { sendTemplate } = require("../sendinblue");
+const { SENDINBLUE_TEMPLATES } = require("snu-lib");
+const { ADMIN_URL } = require("../config");
+
+const { ROLES, departmentLookUp, department2region, MISSION_DOMAINS } = require("snu-lib");
+const { updateApplication } = require("../utils/index.js");
+
+let startTime = new Date();
+
+const jva2SnuDomaines = {
+  "Mobilisation Covid19": MISSION_DOMAINS.SOLIDARITY,
+  "Éducation pour tous": MISSION_DOMAINS.EDUCATION,
+  "Santé pour tous": MISSION_DOMAINS.HEALTH,
+  "Protection de la nature": MISSION_DOMAINS.ENVIRONMENT,
+  "Solidarité et insertion": MISSION_DOMAINS.SOLIDARITY,
+  "Solidarité & Insertion": MISSION_DOMAINS.SOLIDARITY,
+  "Sport pour tous": MISSION_DOMAINS.SPORT,
+  "Prévention et protection": MISSION_DOMAINS.SECURITY,
+  "Mémoire et citoyenneté": MISSION_DOMAINS.CITIZENSHIP,
+  "Coopération internationale": MISSION_DOMAINS.CITIZENSHIP,
+  "Art et culture pour tous": MISSION_DOMAINS.CULTURE,
+  "Art & Culture pour tous": MISSION_DOMAINS.CULTURE,
+};
+
+const fetchMission = (page = 1) => {
+  fetch(`https://www.jeveuxaider.gouv.fr/api/api-engagement/missions?filter[is_snu_mig_compatible]=1&apikey=${JVA_API_KEY}&page=${page}`, {
+    method: "GET",
+    redirect: "follow",
+  })
+    .then((response) => response.json())
+    .then((result) => sync(result))
+    .then((rest) => (rest ? fetchMission(++page) : cleanData()))
+    .catch((error) => console.log("error fetch mission :", error));
+};
+
+const fetchStructure = async (id) => {
+  return fetch(`https://www.jeveuxaider.gouv.fr/api/api-engagement/organisations/${id}?apikey=${JVA_API_KEY}`, {
+    method: "GET",
+    redirect: "follow",
+  })
+    .then((response) => response.json())
+    .catch((error) => console.log("error fetch struture :", error));
+};
+
+const sync = async (result) => {
```

Comment:
Nickel c'est super propre !

Normalement on est sensé valider toutes les données en provenance de JVA vu que c'est de la données extérieure, mais on n'a qu'à dire que ce sera dans une prochaine itération.